include/exfat_ondisk.h

```diff
@@ -60,6 +60,8 @@
 #define EXFAT_STREAM		0xC0	/* stream entry */
 #define EXFAT_NAME		0xC1	/* file name entry */
 #define EXFAT_ACL		0xC2	/* stream entry */
+#define EXFAT_VENDOR_EXT	0xD0
+#define EXFAT_VENDOR_ALLOC	0xD1
```

Comment:
Hi @hclee  @namjaejeon 

EXFAT_VENDOR_EXT should be 0xE0 and EXFAT_VENDOR_ALLOC should be 0xE1.

Reply:
Right. @hclee will fix it. Let me know if you have more review points.